src/state/opamEnv.ml

```diff
@@ -639,6 +639,8 @@ let update_user_setup root ?dot_profile shell =
     OpamStd.Option.iter (fun f -> update_dot_profile root f shell) dot_profile
   )
 
+let check st = is_up_to_date ~check:true st
```

Comment:
I would remove this function, it seems it's no longer needed (and `is_up_to_date` has a clearer name)